src/runtime/entitlements-manager-test.js

```diff
@@ -39,15 +40,16 @@ describes.realWin('EntitlementsManager', {}, env => {
   let storageMock;
   let config;
   let analyticsMock;
+  let eventManagerMock;
   let deps;
   let encryptedDocumentKey;
-  let eventManager;
 
   beforeEach(() => {
     win = env.win;
     pageConfig = new PageConfig('pub1:label1');
     fetcher = new XhrFetcher(win);
-    eventManager = new ClientEventManager();
+    const eventManager = new ClientEventManager();
```

Comment:
I'm slightly surprised this works through testing without passing Promise.resolve() to ClientEventManager as the first parameter.  For robustness it would be best to do so.

Reply:
I believe it works because I am mocking out calls to logEvent, which is the function in ClientEventManager that uses the promise. In any case, I'll add it for robustness.